source/Core/Controller/BaseController.php

```diff
@@ -423,9 +430,12 @@ public function getViewData()
      */
     public function getViewDataElement($sParamId = null)
     {
+        $return = null;
         if ($sParamId && isset($this->_aViewData[$sParamId])) {
-            return $this->_aViewData[$sParamId];
+            $return = $this->_aViewData[$sParamId];
         }
+
+        return $return;
```

Comment:
I don't understand, what you're trying to do here, since the return value is already `null` if you don't return any other value:
```
Psy Shell v0.9.9 (PHP 7.2.10-0ubuntu0.18.04.1 — cli) by Justin Hileman
>>> function foo() { }
>>> function bar() { return; }
>>> function baz() { return null; }
>>> foo();
=> null
>>> bar();
=> null
>>> baz();
=> null
```

Same goes for the changes in `public function getSidForWidget()`

Reply:
i got a inspection information on my IDE:
Missing return statement.
Inspection info:
The inspection is intended to point at the following types of inconsistencies:
- The function/method contains return statements with arguments as well as without them
- The function/method may happen to return a value or end its execution without a return statement at all
- The function/method is declared void but contains return statements with arguments

Technically these are not errors but practically usually indicate programmer’s mistakes.

Should I remove the changes in both methods?